tests/compatibility-tests/pom.xml

```diff
@@ -337,6 +337,8 @@
                               <arg>org.apache.activemq:artemis-server:${project.version}</arg>
                               <arg>org.apache.activemq:artemis-journal:${project.version}</arg>
                               <arg>org.apache.activemq:artemis-hornetq-protocol:${project.version}</arg>
+                              <arg>org.apache.activemq:artemis-jakarta-openwire-protocol:${project.version}</arg>
+                              <arg>org.apache.logging.log4j:log4j-slf4j2-impl:2.21.1</arg>
```

Comment:
This should use the version property to pick up the appropriate version the codebase is using.

Reply:
Yes, indeed.